logicrunner/handle_calls.go

```diff
@@ -158,6 +157,30 @@ func (h *HandleCall) handleActual(
 	if objRef == nil {
 		return nil, errors.New("can't get object reference")
 	}
+	if !objRef.Record().Equal(reqInfo.ObjectID) {
+		return nil, errors.New("object id we calculated doesn't match ledger")
+	}
+
+	repl := &reply.RegisterRequest{Request: *requestRef}
```

Comment:
repl should be replyRequest

Reply:
why? why not requestRegisteredReply?

Reply:
because it's not obvious what repl is later on, maybe it's just me

(both this and that for me repl is Read-eval-print loop)

Reply:
renamed